cvat/apps/engine/views.py

```diff
@@ -2644,7 +2633,7 @@ class AssetsViewSet(
     queryset = Asset.objects.select_related(
         'owner', 'guide', 'guide__project', 'guide__task', 'guide__project__organization', 'guide__task__organization',
     ).all()
-    parser_classes=_UPLOAD_PARSER_CLASSES
+    parser_classes = api_settings.DEFAULT_PARSER_CLASSES + [MultiPartParser]
```

Comment:
```suggestion
    parser_classes = [MultiPartParser]
```

`FileField` doesn't work with JSON, and AFAICS there aren't any other endpoints that take request bodies, so you might as well disable JSON entirely.